frontend/src/components/panels/NodeGraph.svelte

```diff
@@ -497,18 +529,27 @@
 		}}
 	>
 		{#if nodeListLocation}
-			<LayoutCol class="node-list" data-node-list styles={{ "margin-left": `${nodeListX}px`, "margin-top": `${nodeListY}px` }}>
+			<LayoutCol
+				class="node-list"
+				data-node-list
+				styles={{ left: `${nodeListX}px`, top: `${nodeListY}px`, transform: `translate(${appearRightOfMouse ? -100 : 0}%, ${appearAboveMouse ? -100 : 0}%)` }}
+			>
 				<TextInput placeholder="Search Nodes..." value={searchTerm} on:value={({ detail }) => (searchTerm = detail)} bind:this={nodeSearchInput} />
-				{#each nodeCategories as nodeCategory}
-					<LayoutCol>
-						<TextLabel>{nodeCategory[0]}</TextLabel>
-						{#each nodeCategory[1] as nodeType}
-							<TextButton label={nodeType.name} action={() => createNode(nodeType.name)} />
-						{/each}
-					</LayoutCol>
-				{:else}
-					<TextLabel>No search results</TextLabel>
-				{/each}
+				<div style="height: 200px; overflow-y: scroll;" on:wheel|stopPropagation>
+					{#each nodeCategories as nodeCategory}
+						<details style="display: flex; flex-direction: column;" open={nodeCategory[1].shouldBeOpen}>
```

Comment:
We generally prefer adding styles to the css rather than inline.

Reply:
Ok, good to know. I was thinking since was only two it doesn't make sense to make a class, but this does avoid confusion of CSS in two places.